lib/mix/tasks/install.ex

```diff
@@ -136,14 +136,16 @@ defmodule Mix.Tasks.Lvn.SwiftUI.Install do
   end
 
   defp run_xcodegen(%{app_namespace: app_namespace, native_path: native_path}, %{targets: targets}) do
-    lvn_ios_include_path = if "iOS" in targets, do: "project_ios.yml", else: "skip_spec.yml"
-    lvn_macos_include_path = if "macOS" in targets, do: "project_macos.yml", else: "skip_spec.yml"
+    lvn_ios = if "iOS" in targets, do: "iOS", else: ""
+    lvn_macos = if "macOS" in targets, do: "macOS", else: ""
+    lvn_tvos = if "tvOS (Experimental)" in targets, do: "tvOS", else: ""
     lvn_watchos_include_path = if "watchOS" in targets, do: "project_watchos.yml", else: "skip_spec.yml"
     xcodegen_env = [
       {"LVN_APP_NAME", app_namespace},
-      {"LVN_BUNDLE_IDENTIFIER", "com.myorg.#{app_namespace}"},
-      {"LVN_IOS_INCLUDE_PATH", lvn_ios_include_path},
-      {"LVN_MACOS_INCLUDE_PATH", lvn_macos_include_path},
+      {"LVN_BUNDLE_IDENTIFIER", "com.example.#{app_namespace}"},
```

Comment:
I've seen Apple recommend the `com.example.*` naming, so I changed this. It might be better to make this configurable during setup though, or infer it from the base URL.